File: sashaAndSticks.cpp
#include<iostream>
using namespace std;

int main(){
	long long n,k;
	cin >> n >>k;

	long games = n/k;
	if(games % 2 !=0){
		cout<< "YES";
	}else{
		cout<< "NO";
	}
	return 0;
}

File: sashaAndSticks.js
var sashaAndSticks = (n,k)=>{
	var games = Math.floor(n/k);
	if(games % 2 !==0){
		return 'YES'
	}else{
		return 'NO'
	}

}

console.log(sashaAndSticks(1,1)) //YES
console.log(sashaAndSticks(4,1)) // NO
console.log(sashaAndSticks(4,3)) // YES
console.log(sashaAndSticks(10,4))// NO
console.log(sashaAndSticks(7,3)) // NO
console.log(sashaAndSticks(251656215122324104,164397544865601257)) //YES
